site/content/en/docs/contrib/testing.en.md

```diff
@@ -6,17 +6,35 @@ description: >
   How to run tests
 ---
 
-### Unit Tests
+## Prerequisites
+
+* Go distribution
+  * Specific version depends on minikube version.
+  * The current dependency version can be found here : [master branch's go.mod file](https://github.com/kubernetes/minikube/blob/master/go.mod).
+* Since unit test need kvm2 driver, you may need to install `libvirt-dev`:
```

Comment:
This requirement is only specific for Linux: on Windows or macOS, libvirt is not required.

Reply:
Thanks, I agree.